components/CollectivePicker.js

```diff
@@ -35,8 +35,8 @@ const CollectiveTypesI18n = defineMessages({
     defaultMessage: '{n, plural, one {person} other {people}}',
   },
   [CollectiveType.VENDOR]: {
-    id: 'collective.types.vendor',
-    defaultMessage: '{n, plural, one {Vendor} other {Vendors}}',
+    id: 'CollectiveType.Vendor',
```

Comment:
Unified variable name with the other one so that contributors won't have 2 different strings to translate